src/plugins/intel_cpu/src/utils/debug_capabilities.h

```diff
@@ -15,12 +15,71 @@
 
 #include <onednn/dnnl.h>
 #include <dnnl_debug.h>
+#include <cpu/x64/jit_generator.hpp>
 
 #include "openvino/core/model.hpp"
 
 namespace ov {
 namespace intel_cpu {
 
+class RegPrintManager;
```

Comment:
There is no much sense to put this functionality inside major debug capabilities file, since it shouldn't depend on DEBUG_CAPS option. Lets move inside smt like  `utils.hpp` inside emitters folder.